.github/ISSUE_TEMPLATE/os1-bug-report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: OS1 Bug report
+about: 'Use this template to file a bug for any OS1 app: Chat, Publish, Links, Groups,
+  Weather or Clock'
+title: ''
+labels: OS1
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```

Comment:
Usually we add "actual behaviour" too

Reply:
I felt like this was covered under repro.